### DISCUSSION TITLE
Draggable sprite not draggable on Leopard stage

### DISCUSSION THREAD

**Discussion by sfederici** [leopard-js/leopard-mentors#5] [Cleanup Crew]
### Scratch Project URL

https://scratch.mit.edu/projects/992215015/

### Leopard Project URL

https://leopardjs.com/projects/06O9Xowsoj

### What kind of error(s) are you experiencing?

- [ ] Leopard is failing to generate JavaScript code
- [X] The JavaScript code generated by Leopard doesn't work correctly
- [ ] The JavaScript code generated by Leopard is confusing or needs improvement
- [ ] Other

### Description of issue(s)

The sun in Scratch is draggable, but in the Leopard stage it is not

**Comment by towerofnix** [comment]
Right, "set drag mode" currently isn't supported: leopard-js/leopard#123

It and "stop all" are pretty much the only blocks we don't have an implementation for, yet.

There isn't any workaround that's near as nice as "set drag mode" — we should add support on Leopard's end, soon!

**Comment by sfederici** [comment]
This is the usual workaround

<img width="423" alt="immagine" src="https://github.com/leopard-js/leopard-mentors/assets/2651868/db87afd0-ab4b-4d23-8085-3175b2018e20">
